### DISCUSSION TITLE
JBD BMS connected via Bluetooth

### DISCUSSION THREAD

**Discussion by Moach2000** [Louisvdw/dbus-serialbattery#935] [Q&A] [answered]
Dear Community!
As I know from dbus (from the description), there should be the possibility to connect a JBD BMS via Bluetooth.
Question - how?

I use a Raspberry 3B (with Bluetooth of course) and installed Venus OS (latest version from Dec. 2023 or similar), used Putty via SSH to install dbus-serial (via script).
If I scan the environment for the BMS, of course I can see it in the listed devices.
DVCC should be activated, otherwise Bluetooth isn't possible?!

Can someone support me?

--> Bluetooth with JBD BMS possible?
--> what's the missing step within me - above mentioned - way of installing and using Venus OS.

Best regards

**Comment by AlexdeKairos** [comment, marked as answer]
Hi @Moach2000 ,

Yes, it's possible.

Have a look to the installation instructions.  You'll notice that for BT installation, you must notify via config.ini file, your battery MAC address.

Review this help:
https://louisvdw.github.io/dbus-serialbattery/general/install

After editing the file, you need to reinstall the driver again (or reactivate)

In the help, you have the SSH commands to do it. The installer will show you how many BLE BMSs are found and installed.

If the distance between your batteries and venus device is high, maybe you need an external Bluetooth. 
With your installation,  you also received a sample config.ini with the different options available to match your setup.

Enjoy 
Alex

**Reply by Moach2000** [reply]
Hi Alex! I'll try and in case of issues - I'll come back :) Br, Chris

**Reply by Moach2000** [reply]
Update - I ordered a RS485 adapter (instead using BLE anymore without any success). Via RS485, I'm able to see values.

**Reply by AlexdeKairos** [reply]
Hi. One think I could fond on JBD BLE connection is that actually I could only use dbus-fast between 1.94.0 to 1.95 for dual battery and dual BMS. In my particular setup. I had to install an external Bluetooth dongle. Sad it didn't work for you but cable is more reliable than Bluetooth atm. 
Good luck!
